www/content/docs/components/BarChart.mdx

```diff
@@ -0,0 +1,134 @@
+---
+name: BarChart
+---
+
+<Playground showEditor={true}>
```

Comment:
I'm getting a `ReferenceError: BarChart is not defined` when I open the deploy preview for this PR and navigate to the `Bar Chart` docs. Not sure if it's just the PR environment or an actual issue?

Reply:
This is because of the way the docs are set up. They currently load the latest version of palette published on npm, rather than the local version. I'm working on a fix for this (moving palette repo to yarn workspaces) but it'll happen after this PR.

Reply:
Ah! Okay cool. 👍